<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>omni::ui::NewCanvas</class>
 <widget class="QDialog" name="omni::ui::NewCanvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Projection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="verticalFrame">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="horizontalFrame">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="canvasLabel">
              <property name="text">
               <string>Canvas:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbCanvasType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="canvasFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="horizontalFrame_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="projectorSetupLabel">
              <property name="text">
               <string>Projector Setup:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbProjSetup"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="projSetupFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Preview</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="omni::ui::PreviewGL" name="previewGL" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnNew">
        <property name="text">
         <string>Create new projection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>omni::ui::PreviewGL</class>
   <extends>QWidget</extends>
   <header>omni/ui/PreviewGL.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnNew</sender>
   <signal>clicked()</signal>
   <receiver>omni::ui::NewCanvas</receiver>
   <slot>newProjection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>omni::ui::NewCanvas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCanvasType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>omni::ui::NewCanvas</receiver>
   <slot>canvasChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbProjSetup</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>omni::ui::NewCanvas</receiver>
   <slot>projSetupChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newProjection()</slot>
  <slot>canvasChanged(int)</slot>
  <slot>projSetupChanged(int)</slot>
 </slots>
</ui>
